src/ReactiveUI.XamForms/ViewModelViewHost.cs

```diff
@@ -70,7 +89,9 @@ public ViewModelViewHost()
                 (vm, contract) => new { ViewModel = vm, Contract = contract, });
 
             var platform = Locator.Current.GetService<IPlatformOperations>();
-            if (platform == null) {
+
+            if (platform == null)
+            {
```

Comment:
Good catch. My Rebracer was playing up. I've just restarted VS and re-applied, then pushed another commit.